packages/tic-tac-toe-bot/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@counterfactual/tic-tac-toe-bot",
+  "version": "0.0.1",
+  "description": "A bot for the Tic Tac Toe Playground dApp.",
+  "author": "Counterfactual",
+  "homepage": "https://github.com/counterfactual/monorepo",
+  "license": "MIT",
+  "main": "src/index.js",
```

Comment:
Isn't the main file a Typescript file? Or will we build and distribute? In which case it would be `dist/index.js`